e2e/explorer/api/src/app/etherscan/types.ts

```diff
@@ -58,6 +58,16 @@ export interface TxListInternal extends BaseParams<'account', 'txlistinternal'>
   /** the sorting preference, use asc to sort by ascending and desc to sort by descendin Tip: Specify a smaller startblock and endblock range for faster search results. */
   sort: Sort;
 }
+export interface BlockMined extends BaseParams<'account', 'blockMined'> {
+/** the string representing the address to check for balance */
+address: string;
+/** the string pre-defined block type, either blocks for canonical blocks or uncles for uncle blocks only */
+blocktype?: string;
```

Comment:
Based on the comment it should be 
```suggestion
blocktype?: 'blocks' | 'uncles';
```